src/deepforest/model.py

```diff
@@ -403,3 +422,58 @@ def val_dataset_confusion(self, return_images=False):
             return images, true_label, predicted_label
         else:
             return true_label, predicted_label
+
+    @classmethod
+    def load_model(
+        cls,
+        repo_id,
+        revision=None,
+    ):
+        """Load a model from the Hugging Face Hub.
+
+        Args:
+            repo_id: Hugging Face repo id, e.g. "username/my-cropmodel".
+            revision: Optional git revision/branch/tag. Defaults to repo default.
+
+        Returns:
+            CropModel: The loaded and eval-mode model instance.
+        """
+
+        model = cls.from_pretrained(
+            repo_id,
+            revision=revision,
+        )
+        model.eval()
+
+        return model
+
+    @classmethod
+    def from_config(cls, config):
+        """Recreate instance from Hub config (used by from_pretrained)."""
+        num_classes = config.get("num_classes")
+        label_dict = config.get("label_dict")
+
+        instance = cls(
+            num_classes=num_classes,
+            batch_size=config.get("batch_size", 4),
+            num_workers=config.get("num_workers", 0),
+            lr=config.get("lr", 0.0001),
+            label_dict=label_dict,
+            model=None,
+        )
+
+        # Ensure underlying model is created with correct number of classes
+        if num_classes is not None and getattr(instance, "model", None) is None:
+            instance.create_model(num_classes)
+
+        # Keep reverse mapping
+        if label_dict is not None:
+            instance.numeric_to_label_dict = {v: k for k, v in label_dict.items()}
+
+        instance.update_config()
+        return instance
```

Comment:
I'm unclear on what this code is doing. I tried testing locally without it and the tests still seem to pass (though I was rushing so I could have messaged something up)

Reply:
Will remove.